ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -81,6 +81,7 @@ data TraceDb
   | TDBValueHandle Int TraceValueHandle
   | TDBTableOp     TraceTableOp
   | TDBInitialisingFromLMDB !FS.FsPath
+  | TDBInitialisingFromLMDBDone !FS.FsPath
```

Comment:
We don't use `Done` in the events above, but the past tense of the verb instead. Maybe we want to do the same here, ie `TDBInitialisedFromLMDB`